modules/client-common/src/types.ts

```diff
@@ -40,6 +40,15 @@ export enum SupportedNetwork {
 
 export const SupportedNetworksArray = Object.values(SupportedNetwork);
 
+
+export enum SupportedVersion {
+  V1_0_0 = "v1.0.0",
+  V1_3_0 = "v1.3.0",
+  LATEST = "v1.3.0",
```

Comment:
```suggestion
  LATEST = SupportedVersion.V1_3_0,
```
Suprisingly, this works, as you can [check on the TS playground](
https://www.typescriptlang.org/play?#code/KYOwrgtgBAymAO8D2AnALsAJgNWCgzgJZIhQDeAUFFNgIwD6ADE1ALxQBEAbrQHSP8OAGio0GAZhbtufcYJHUAMgEEAKgFEYqtrATJ0WXAWIhedepMYiAvhQoBjEviQAbYLxdIA5gAoO55kYALmE4RFQMHDwiEjMGQIBKBydXd09ffwkmEKEw-UijGNNzSyTHEGc3D28-FQ0tHN1wgyjjWLrNVQSgA)

Reply:
Yep, TS does not complain but it chrashes when I try to build it